lms/djangoapps/teams/static/teams/js/spec/views/teams_tab_spec.js

```diff
@@ -44,6 +75,12 @@ define([
         });
 
         describe('Navigation', function () {
+            it('does not interfere with anchor links to #content', function () {
```

Comment:
Aesthetically and conceptually it should be separate from the `'displays and focuses an error message *'` test cases.